lib/active_model/type/salesforce/multipicklist.rb

```diff
@@ -3,20 +3,27 @@
 module ActiveModel
   module Type
     module Salesforce
-      class Mulitpicklist < ActiveModel::Type::Value
+      class Multipicklist < ActiveModel::Type::Value
+        include ActiveModel::Type::Helpers::Mutable
 
         def type
           :multipicklist
         end
 
-        private
+        def deserialize(value)
+          value.to_s.split(';')
```

Comment:
Set is probably a better representation of Multiselect Picklist, but I thought it would be better not to change it to prevent any negative outcome.